File: Cpp/Basics/CF_110_A.cpp
#include <iostream>
#include <string>

int main()
{
    std::ios_base::sync_with_stdio(false);
    std::cin.tie(NULL);

    std::string x;

    std::cin >> x;

    unsigned long long counter{0};
    for (char c : x)
    {
        int x = c - '0';
        if (x == 4 || x == 7)
        {
            counter++;
        }
    }
    std::string y = std::to_string(counter);

    bool lucky = true;
    for (char c : y)
    {
        if (c != '4' && c != '7')
        {
            lucky = false;
            break;
        }
    }
    if (lucky)
    {
        std::cout << "YES\n";
    }
    else
    {
        std::cout << "NO\n";
    }

    return EXIT_SUCCESS;
}

File: Julia/Basics/CF_110_A.jl

# Not Working Properly
function solve()
    x = readline()
    counter = 0
    for i in x
        j = i - '0'
        if j==4 || j==7
            counter += 1
        end
    end
    y = parse(string,counter)
    lucky = true
    for i in y
        if i!= '4' && c!='7'
            lucky = false
            break
        end
    end
    if lucky
        println("YES")
    else
        println("NO")
    end
end

solve()